CONTRIBUTING.md

```diff
@@ -165,3 +165,27 @@ general guidelines for the community as a whole:
   respond to an email you are potentially sending to a large number of
   people.  Please consider this before you update.  Also remember that
   nobody likes spam.
+
+## FAQ
+
+**Is it fine to go with kotlin for my contribution?**
+
+**Is there an architectural design for the presentation layer?**
+
+**Is there an architectural design for the data layer?**
+
+**Where does the sitemap rendering happen?**
+
+**How is navigation managed?**
```

Comment:
Please clarify that question; I'm not sure what the actual question is here.

Reply:
Is navigation managed in one place? Like for example the new architecture navigation component.

Reply:
No. ContentController manages how fragments are displayed, and MainActivity commands it what fragments to display. Navigation is too dynamic in our app (determined by sitemap) for the navigation architecture component to be useful.